prody/proteins/pdbfile.py

```diff
@@ -1032,11 +1032,6 @@ def _evalAltlocs(atomgroup, altloc, chainids, resnums, resnames, atomnames):
                         'atomgroup {1}.'.format(repr(key), atomgroup.getTitle()))
             atomgroup.addCoordset(xyz, label='altloc ' + key)
 
-PDBLINE = ('{0:6s}{1:5d} {2:4s}{3:1s}'
-           '{4:4s}{5:1s}{6:4d}{7:1s}   '
-           '{8:8.3f}{9:8.3f}{10:8.3f}'
-           '{11:6.2f}{12:6.2f}      '
-           '{13:4s}{14:2s}\n')
```

Comment:
Why was this deleted? Was it not used at all?

Reply:
Nope. Not anywhere. I thought it was confusing to have it therefore.

Reply:
I just searched and you are right. This was not used anywhere.

Reply:
oh, I guess I should double check that it wasn't imported in any other file

Reply:
ok, yep, that's the only file and it's not used